File: src/main/java/com/jss/osiris/modules/osiris/crm/service/WebinarService.java
package com.jss.osiris.modules.osiris.crm.service;

import java.util.List;

import com.jss.osiris.modules.osiris.crm.model.Webinar;

public interface WebinarService {

    public List<Webinar> getWebinars();

    public Webinar getWebinar(Integer id);

    public Webinar addOrUpdateWebinar(Webinar webinar);

    public Webinar getLastWebinar();

    public Webinar getNextWebinar();
}

File: client-myjss/src/app/modules/tools/services/webinar.service.ts
import { HttpClient, HttpParams } from '@angular/common/http';
import { Injectable } from '@angular/core';
import { AppRestService } from '../../main/services/appRest.service';
import { Webinar } from '../model/Webinar';

@Injectable({
  providedIn: 'root'
})
export class WebinarService extends AppRestService<Webinar> {

  constructor(http: HttpClient) {
    super(http, "crm");
  }

  getLastWebinar() {
    return this.get(new HttpParams(), 'webinar/last');
  }

  getNextWebinar() {
    return this.get(new HttpParams(), 'webinar/next');
  }
}